scripts/travis/04-install-thunder.sh

```diff
@@ -37,7 +37,7 @@ drush_make_thunder() {
 
 composer_create_thunder() {
     cd ${THUNDER_DIST_DIR}
-    composer create-project burdamagazinorg/thunder-project ${TEST_DIR} --stability dev --no-interaction --no-install
+    composer create-project burdamagazinorg/thunder-project:dev-feature/remove-nvm ${TEST_DIR} --stability dev --no-interaction --no-install
```

Comment:
I think we should make new branch + release of thunder-project (8.2.0 I guess) and then use that here instead of branch.